waspc/data/Cli/bash-completion

```diff
@@ -1,23 +1,5 @@
-_wasp()
-{
-    
-    local cur prev
-    cur="${COMP_WORDS[COMP_CWORD]}"
-    prev="${COMP_WORDS[COMP_CWORD-1]}"
-
-    case ${COMP_CWORD} in
-        1)
-            # auto complete for wasp <command>
-            _wasp_commands=$( wasp completion:list )
-            COMPREPLY=( $(compgen -W "${_wasp_commands}" -- "${cur}") )
-            ;;
-        2)
-            # auto complete for subcommand wasp <command> <subcommand>
-            _wasp_second_commands=$( wasp completion:list ${prev} )
-            COMPREPLY=( $(compgen -W "${_wasp_second_commands}" -- "${cur}") )
-            ;;
-        *)
-            COMPREPLY=()
-    esac
+function _wasp {
+    wasp completion:list
```

Comment:
Ah nice, this is now super short, all logic is in Haskell and it is hard to imagine users will ever need to update this file! Awesome.

Reply:
Yep, this was quite a good learning exercise for me too on how bash autocompletion works. Even cooler that it exposes this functionality for externally supplying completion via the binary :)

Reply:
Yes, same for me :)! They did really nice job with this.